monitoring/prober/rid/test_isa_simple.py

```diff
@@ -83,34 +92,18 @@ def test_update_isa(session):
                   'altitude_lo': 20,
                   'altitude_hi': 400,
               },
-              'time_start': time_start.strftime(rid.DATE_FORMAT),
-              'time_end': time_end.strftime(rid.DATE_FORMAT),
           },
-          'flights_url': 'https://example.com/dss',
+          'flights_url': 'https://example.com/dss/v2',
       })
   assert resp.status_code == 200
 
   data = resp.json()
   assert data['service_area']['id'] == ISA_ID
-  assert data['service_area']['flights_url'] == 'https://example.com/dss'
-  assert data['service_area']['time_start'] == time_start.strftime(
-      rid.DATE_FORMAT)
-  assert data['service_area']['time_end'] == time_end.strftime(
-      rid.DATE_FORMAT)
+  assert data['service_area']['flights_url'] == 'https://example.com/dss/v2'
   assert re.match(r'[a-z0-9]{10,}$', data['service_area']['version'])
   assert 'subscribers' in data
 
 
-@default_scope(SCOPE_READ)
-def test_get_isa_by_id(session):
```

Comment:
I think it's a good idea to get the ISA above, but I think we want to keep this test here also -- it would be great to validate the actual change (flights_url -> https://example.com/dss/v2)